api/v1/group.go

```diff
@@ -84,63 +82,6 @@ func NewGroup(g *lib.Group, parent *Group) *Group {
 	return group
 }
 
-// GetID gets a group ID
-// Deprecated: use instead g.ID directly
-// This method will be removed with the one of the PRs of (https://github.com/grafana/k6/issues/911)
-func (g Group) GetID() string {
-	return g.ID
-}
-
-// SetID sets a group ID
-// Deprecated: use instead g.ID directly
-// This method will be removed with the one of the PRs of (https://github.com/grafana/k6/issues/911)
-func (g *Group) SetID(v string) error {
-	g.ID = v
-	return nil
-}
-
-// GetReferences returns the slice of jsonapi.References
-// Deprecated: use instead g.Groups properties
```

Comment:
If we decide to keep the deprecation for a version it seems to me like `// Deprecated` needs to be the last comment and separated with a newline from the rest in order for it to work ... but I might be wrong.